sources/packages/backend/apps/queue-consumers/src/processors/schedulers/cra-integration/cra-response-integration.scheduler.ts

```diff
@@ -5,14 +5,14 @@ import { QueueNames } from "@sims/utilities";
 import { InjectLogger, LoggerService } from "@sims/utilities/logger";
 import { Job, Queue } from "bull";
 import { BaseScheduler } from "../base-scheduler";
-import { ProcessResponseQueueOutDTO } from "./models/process-response.dto";
+import { ProcessResponseQueue } from "./models/process-response.dto";
```

Comment:
The new summary interface can be used instead.

Reply:
CRA is part of old story. maybe we can create technical debt for it. we will also need to change IER12 files too